_episodes/05-Fixtures.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Fixtures
+teaching: 10
+exercises: 0
+questions:
+- "How do I create and cleanup the data I need to test the code?"
+objectives:
+- Understand how test fixtures can help write tests.
+keypoints:
+-   It may be necessary to set up "fixtures" composing the test environment.
+---
+The above example didn't require much setup or teardown. Consider, however, the
+following example that could arise when comunicating with third-party programs.
+You have a function `f()` which will write a file named `yes.txt` to disk with
+the value 42 but only if a file `no.txt` does not exist. To truly test that the
+function works, you would want to ensure that neither `yes.txt` nor `no.txt`
+existed before you ran your test. After the test, you want to clean up after
+yourself before the next test comes along.  You could write the test, setup,
+and teardown functions as follows:
+
+~~~
```

Comment:
I don't like this example, because it doesn't connect to the previous exercise. I thought we had some better ideas here